apps/registration/src/utils/fhirPatientMapper.ts

```
@@ -225,3 +232,168 @@ export function buildFhirPatient(input: MapperInput): Patient {
+          middleName: fhirName.given?.[1],
+          familyName: fhirName.family ?? '',
+          preferred: true,
+          voided: false,
+        },
+      ]
+    : [];
+
+  // ── Birthdate / estimated / birthtime ────────────────────────────────────
+  const rawBirthDate = fhirPatient.birthDate ?? '';
+  const birthdateEstimated =
+    rawBirthDate.length === 7; // 'YYYY-MM' implies estimated
```

ESLint

[error] Delete `⏎···`

(prettier/prettier)